packages/analytics-js-integrations/src/integrations/Braze/browser.js

```diff
@@ -66,6 +68,16 @@ class Braze {
     window.braze.openSession();
   }
 
+  isLoaded() {
+    logger.debug('In isLoaded');
+    return this.appKey && window.brazeQueue === null;
```

Comment:
`this.appKey` in this logic does not make sense.

Reply:
Same in `isReady`

Reply:
updated